website/docs/d/psuinstpol.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+subcategory: -
+layout: "aci"
+page_title: "ACI: aci_power_supply_redundancy_policy"
+sidebar_current: "docs-aci-data-source-power_supply_redundancy_policy"
+description: |-
+  Data source for ACI Power Supply Redundancy Policy
+---
+
+# aci_power_supply_redundancy_policy #
+
+Data source for ACI Power Supply Redundancy Policy
+
+
+## API Information ##
+
+* `Class` - psuInstPol
+* `Distinguished Name` - uni/fabric/psuInstP-{name}
+
+## GUI Information ##
+
+* `Location` - Fabric -> Fabric Policies -> Policies -> Switch -> Power Supply Redundancy
+
+
+## Example Usage ##
+
+```hcl
+data "aci_power_supply_redundancy_policy" "example" {
+
+  name  = "example"
+}
+```
+
+## Argument Reference ##
+
+* `name` - (Required) Name of the Power Supply Redundancy Policy object.
+
+## Attribute Reference ##
+* `id` - Attribute id set to the Dn of the Power Supply Redundancy Policy.
+* `annotation` - (Optional) Annotation of the Power Supply Redundancy Policy object.
+* `name_alias` - (Optional) Name Alias of the Power Supply Redundancy Policy object.
+* `admin_rdn_m` - (Optional) Admin Redundancy Mode. The administrative state of the power supply policy.
```

Comment:
When data sources are set to computed they should be changed to Read-Only

```suggestion
* `annotation` - (Read-Only) Annotation of the Power Supply Redundancy Policy object.
* `name_alias` - (Read-Only) Name Alias of the Power Supply Redundancy Policy object.
* `admin_rdn_m` - (Read-Only) Admin Redundancy Mode. The administrative state of the power supply policy.
```

Reply:
I know `annotation` and `name_alias` also have optional set from generic function but please set Read-Only in docs because they will be overwritten regardless. We should in time probably change the annotation/name_alias attribute adds for all data-sources.